docs/pages/guides/extending-world.mdx

```diff
@@ -484,3 +488,567 @@ The easiest way to create the Solidity code is to use the MUD template:
    The next 320 characters are words 0-4, which are not the actual message (each word is 32 bytes, which is 64 hexadecimal digits). We also don't need the trailing newline. `${RAW_RESULT:322:-2}` removes those characters so we can use [`cast`](https://book.getfoundry.sh/reference/cast/cast-to-ascii) to get the ASCII.
 
    </details>
+
+## Create an updated user interface
+
+Finally, we create a user interface that supports the messaging system.
+
+### Run the user interface
+
+For the user interface to run properly, it cannot require resources that the template user interface is already using, or that are not available because of the changes we made in `packages/contracts`.
+Here we modify the template UI to so it will run properly.
+
+1. Change to the client package of the extension.
+
+   ```sh copy
+   cd ../client
+   ```
+
+1. Port 3000 is already in use by the `extendMe` application, so edit `vite.config.ts` to use a different port.
+
+   <CollapseCode>
+
+   ```ts filename="vite.config.ts" copy showLineNumbers {5}
+   import { defineConfig } from "vite";
+
+   export default defineConfig({
+     server: {
+       port: 3001,
+       fs: {
+         strict: false,
+       },
+     },
+     build: {
+       target: "es2022",
+       minify: true,
+       sourcemap: true,
+     },
+   });
+   ```
+
+   </CollapseCode>
+
+1. Start the user interface.
+
+   ```sh copy
+   pnpm vite
+   ```
+
+1. Open the application at [http://localhost:3001](http://localhost:3001).
+   At this point the server gives us web pages, but the JavaScript in them does not work, the next steps are to fix it.
```

Comment:
```suggestion
   The client will throw errors because we didn't update the client code to work with the new contracts yet - we'll fix that in the next steps.
```